### DISCUSSION TITLE
PCAP job's stuck at pending

### DISCUSSION THREAD

**Discussion by Mastadamus** [Security-Onion-Solutions/securityonion#4212] [Unsupported Versions] [answered]
I have a standalone centos iso based build updated via soup to 2.3.50. Currently, No pcap jobs are completing whether generated from kibana or the SOC. They stay in "pending" status. I have checked the sensoroni and sensonroni-server logs. For the sensoroni logs the tail CMD shows no errors recently. I can "grep" errors  like so.
![FailedtoPoll](https://user-images.githubusercontent.com/79027361/118533170-5bf25c80-b70d-11eb-97fc-fb1b64ed58fa.JPG)
There are no new errors after this grouping.
I can find no errors in the sensoroni-server log.

I did notice that in the "PCAP" page in the SOC, my ID shows SecOnion.  In the grid page, my id shows seconion. Could this be a name issue caused by that hotfix?

**Comment by jertel** [comment]
I suspect it is, and we're looking into it. Thanks for the details.

**Reply by jertel** [reply]
Tracking in #4220.

**Reply by cyberkimh** [reply]
Yes we're having the same issue. Not been able to view any PCAPs recently.

**Reply by Farcocoop** [reply]
The same issue...

**Comment by TOoSmOotH** [comment, marked as answer]
2.3.51 Should address this. Let me know if you are still seeing this.

**Reply by Mastadamus** [reply]
Works now great. Thank you very much. This was a very fast fix. I appreciate it big league.

**Reply by Farcocoop** [reply]
It works!! Thank you very much!

**Comment by kornflake003-GITHUB** [comment]
Hi, I´m newest in this plataform and have the same problem, where can i find the "sensor ID"?

TNX.

**Reply by tomjones1977** [reply]
If you look in `Grid` you'll see the names of your nodes.  If you have a distributed setup it's the forward node you are looking for.  If you have a standalone node then it's the name of your standalone node.

![image](https://user-images.githubusercontent.com/32241793/143573834-cbfbb02d-4189-4e88-819e-325c3c05ff6b.png)

**Reply by kornflake003-GITHUB** [reply]
Thanks a lot. Last night I solved the problem, but now I have another problem. Tasks went from "pending" to "incomplete" =(.

I generated only 2 tasks to explore the functionality of the PCAP module, but none could be completed successfully.

I really appreciate your support.


![Diapositiva1](https://user-images.githubusercontent.com/85261291/143614709-69bd8aae-7cad-4b67-8bcd-841408191f38.PNG)

![Diapositiva2](https://user-images.githubusercontent.com/85261291/143614728-e5e45aa9-f833-48d2-af11-6b8bf4f7ae4d.PNG)

**Reply by tomjones1977** [reply]
At a guess the PCAP tool isn't happy with the future end date, which is causing it to retry repeatedly and never finish.

Try setting a smaller time window and ensure both dates are in the past, to see if it's a problem with the request itself or a problem with the system.

**Reply by kornflake003-GITHUB** [reply]
Hello, thank you for your support. I have already made several queries with dates ending in the past (start and end), but I still have the same problem. I assume it is a system problem. I include 2 screenshoots for the JOB of the PCAP and the configuration of the GRID.

I appreciate your suppor
![GRID_Conf](https://user-images.githubusercontent.com/85261291/143664017-e3b48fd8-1ecf-4529-bba6-58abe328feb4.png)
t.
![PCAP_Query_NoCompleted](https://user-images.githubusercontent.com/85261291/143664020-5ecf7d46-2b76-4611-a6c1-dc8634cfd7e6.png)

**Reply by tomjones1977** [reply]
Do you definitely have traffic going to port 3389 in that timeframe?  

If you run `tcpdump -n -i XXX` on the CLI (replacing XXX with your sniffing interface name) do you see TAP/SPAN traffic hitting the NIC?

**Reply by dougburks** [reply]
@kornflake003-GITHUB This discussion was marked as answered back in May. From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

> Start a new discussion instead of replying to somebody else's discussion
Please search to see if you can find similar discussions that may help you. However, in order to avoid confusion, please do NOT reply to somebody else's discussion with your own issue. Instead, please start a new discussion and simply provide a hyperlink to the related discussion.

**Reply by kornflake003-GITHUB** [reply]
Ok understood, I am going to generate a new discussion and copy the link of this one. Sorry.

**Reply by kornflake003-GITHUB** [reply]
> Do you definitely have traffic going to port 3389 in that timeframe?
> 
> If you run `tcpdump -n -i XXX` on the CLI (replacing XXX with your sniffing interface name) do you see TAP/SPAN traffic hitting the NIC?

https://github.com/Security-Onion-Solutions/securityonion/discussions/6392

**Reply by kornflake003-GITHUB** [reply]
Hello, I´m created a new discussion:

https://github.com/Security-Onion-Solutions/securityonion/discussions/6392

Thanks for your support.
